README.md

```diff
@@ -823,8 +823,7 @@ Other Style Guides
     ```javascript
     // bad
     [1, 2, 3].map(number => {
-      const nextNumber = number + 1;
-      `A string containing the ${nextNumber}.`;
+      return `A string containing the ${number}.`;
```

Comment:
why this change? The purpose is to make an arrow function that is not simply returning a single expression, and mistakenly omits the `return`.